index.html

```diff
@@ -15,33 +15,23 @@
     <section class="hero is-primary is-medium is-fixed">
         <!-- Hero head: will stick at the top -->
         <div class="hero-head">
-            <nav class="navbar">
+            <div class="navbar">
```

Comment:
so I would move this nav `tag` from here down as this tag is holding too much none nav related